server/const.go

```diff
@@ -175,4 +175,8 @@ const (
 	// DEFAULT_ALLOW_RESPONSE_EXPIRATION is the default time allowed for a given
 	// dynamic response permission.
 	DEFAULT_ALLOW_RESPONSE_EXPIRATION = 2 * time.Minute
+
+	// DEFAULT_SERVICE_LATENCY_SAMPLE is the default sampling rate for service
+	// latency metrics
+	DEFAULT_SERVICE_LATENCY_SAMPLE = 100
```

Comment:
Not sure if this variable has to be named this way. It's not the standard way consts are named in Go. :/

Reply:
Yes I understand, just consistent with how we do it. We started when Go was very young ;)